Bottom: f33ce2d385a77a2550857921b60a4a009a6a2c71
Top:    b91ea658533555ccf386c2388135b28cb5de49fb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-10 23:27:33 -0500

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
